zat.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+if [[ -z "$(docker images -q zat:latest 2> /dev/null)" ]]; then
+  echo "ZAT Image not found, building..."
+  ./scripts/compile.sh
+fi
+
+./scripts/invoke.sh $@
```

Comment:
1. `$@` here also needs to be quoted
2. Does this actually work? If I run this in an app directory, wouldn't it complain something like `./scripts/invoke.sh does not exist`?